Sources/RxAlamofire.swift

```diff
@@ -901,6 +927,53 @@ extension Reactive where Base: DataRequest {
   }
 }
 
+// MARK: Reactive + DownloadRequest
+
+extension Reactive where Base: DownloadRequest {
+    
+  public func response() -> Single<DefaultDownloadResponse> {
+    return Single.create { cb in
+      let request = self.base.response {
+        cb(.success($0))
+      }
+      return Disposables.create(with: request.cancel)
+    }
+  }
+    
+  public func responseSerialized<Serializer: DownloadResponseSerializerProtocol>(
+    queue: DispatchQueue? = nil,
+    responseSerializer: Serializer
+  )
+    -> Single<DownloadResponse<Serializer.SerializedObject>>
+  {
+    return Single.create { cb in
+      let request = self.base.response(
+        queue: queue,
+        responseSerializer: responseSerializer
+      ) {
+        cb(.success($0))
```

Comment:
this is also clever and I'd use it the same in my projects as well, but it seems like all the other methods treat response error as a stream error, not a success... 🤔

Reply:
Hi, @sunshinejr , thanks for the review!

Because the EUR number above is not formatted either, and we already have a label("USD") to indicate what the number below is, I think it is enough to clearly express what we are showing.

So these two values should either be formatted together or not, I can bring it back if you want, 😉.

![fadskjfha](https://user-images.githubusercontent.com/18638838/66793394-cc346d00-ef2f-11e9-8a43-8a5e4b5a6c74.png)